src/components/PrometheusGraph/_Chart.js

```diff
@@ -61,7 +57,7 @@ class Chart extends React.PureComponent {
 
     const cursorValue = valueScale.invert(cursorYOffset);
     const cursorTimestampSec = timeScale.invert(cursorXOffset);
-    const hoverTimestampSec = timestampQuantizer(cursorTimestampSec);
+    const hoverTimestampSec = timestampQuantizer(cursorTimestampSec) || 0;
```

Comment:
I would prefer not to do this - otherwise we say it's ok to show a timestamp `0` which I'd argue is worse than having a warning spit out :)

The proper solution would probably be not to render the timestamp in the tooltip if it's `null` - feel free to do that in `_HoverInfo.js` if you find the warning annoying tho ;)